ansible/roles/chown_cyhy_dirs/tasks/main.yml

```diff
@@ -11,5 +11,5 @@
   loop:
     - /var/cyhy
     - /var/log/cyhy
-  async: 180
+  async: 500
```

Comment:
Did we have a case where this `chown` step was taking more than 3 minutes to complete?

Reply:
Yes, it was sporadically timing out when I applied to my `jsf9k` workspace.  Not often, but occasionally.